test/MultiThreadedCaches.jl

```diff
@@ -120,7 +120,8 @@ end
 
     # Test that all Tasks saw the same exception thrown.
     close(exceptions)
-    @test all_equal(collect(exceptions))
+    output = collect(exceptions)
+    @test all_equal(output)
```

Comment:
Any reason not to add a test, perhaps with your example and `@ballocated`?
e.g. `@test @ballocated foo($c, $10) == 0`

Reply:
i think a test is a _fantastic_ idea!

Reply:
done. thanks!